src/options.rs

```diff
@@ -149,11 +155,14 @@ pub fn builder_from_flags<I>(args: I)
 
     let mut builder = builder();
 
-    if let Some(header) = matches.value_of("header") {
-        builder = builder.header(header);
-    } else {
-        return Err(Error::new(ErrorKind::Other, "Header not found"));
-    }
+    let header = match matches.value_of("header") {
+        Some(h) => h,
+        None => {
+            return Err(Error::new(ErrorKind::Other, "Header not found"));
+        }
+    };
+
+    builder = builder.header(header);
```

Comment:
It seems to me this rewrite isn't necessary for this change, and actually the original way is more compact, and `header` has a clearer life time. I think it'd be better to just keep the original code here.